reactivesocket-core/src/test/java/io/reactivesocket/frame/SetupFrameFlyweightTest.java

```diff
@@ -0,0 +1,30 @@
+package io.reactivesocket.frame;
+
+import io.reactivesocket.FrameType;
+import org.agrona.concurrent.UnsafeBuffer;
+import org.junit.Test;
+
+import java.nio.ByteBuffer;
+
+import static org.junit.Assert.*;
+
+public class SetupFrameFlyweightTest {
```

Comment:
TIOLI: I'd find it really helpful to have at least one hexstring example in the test.